test/tools/libtesteth/TestHelper.cpp

```diff
@@ -501,6 +501,7 @@ void executeTests(const string& _name, const string& _testPathAppendix, const st
 			json_spirit::read_string(s, v);
 			doTests(v, true);
 			addClientInfo(v, testfilename);
+			postParseJson(v);
```

Comment:
doTests adds new sections to json. addClinetInfo should really be inside postParseJson